src/components/Button.js

```diff
@@ -66,9 +67,9 @@ class Button extends React.Component {
 
     return (
       <button
+        className={css(Object.assign({}, styles.component, styles[this.props.type], style))}
```

Comment:
I know the object assign was already there but could you switch it object spread while you're here?

Reply:
👍